src/main/java/uk/gov/justice/digital/delius/controller/secure/OffendersResource.java

```diff
@@ -410,6 +411,23 @@ public OffenderDetail getOffenderDetailByNomsNumber(
                 .orElseThrow(() -> new NotFoundException(String.format("Offender with crn %s not found", crn)));
     }
 
+    @ApiOperation(value = "Return the convictions (AKA Delius Event) for an offender that contain RAR", tags = {"-- Popular core APIs --", "Convictions"})
+    @ApiResponses(
+        value = {
+            @ApiResponse(code = 400, message = "Invalid request", response = ErrorResponse.class),
+            @ApiResponse(code = 404, message = "The offender is not found", response = ErrorResponse.class),
+            @ApiResponse(code = 500, message = "Unrecoverable error whilst processing request.", response = ErrorResponse.class)
+        })
+    @GetMapping(value = "/offenders/crn/{crn}/convictions-with-rar")
+    public List<Conviction> getOffenderConvictionsWithRarByCrn(
```

Comment:
I take your point. It was kept separate as the nature of the search is more in depth, in that it needs to dig into disposals and requirements to determine a match, and it applies to active requirements only. I didn't want to increase the complexity of the existing endpoint by incorporating this functionality. From a consumer's viewpoint it may be better to add it as a separate flag.

Reply:
Sure, I understand. Lets see if we need to add more of these filters in the future and then re-look at it then